app/assets/stylesheets/user.scss

```diff
@@ -0,0 +1,15 @@
+.index-margin {
+  .level {
+    margin-bottom: 0;
+  }
+}
+
+.button-margin {
+  margin-top: 37px;
+}
+
+.column {
+  .select:not(.is-multiple):not(.is-loading)::after {
```

Comment:
THIS IS NOT GOOD
give CLASS to select and specify this style

because you with this change affected ALL selects